### ISSUE TITLE
Flickr embedding plugin

### ISSUE THREAD

**Issue by bolderbrooklyn** [bolderbrooklyn/jbhannah.net-jekyll#3]
Must properly embed medium-sized images from photo ID and link image to photo page on Flickr. Possibly/probably [jekyll-flickr](https://github.com/cnunciato/jekyll-flickr).

**Comment by bolderbrooklyn** [comment]
Fails to build when deploying to Heroku:

```
   bundle exec jekyll
   Build Failed
   Configuration from /tmp/build_g6oesylzhptw/_config.yml
   Building site: /tmp/build_g6oesylzhptw -> /tmp/build_g6oesylzhptw/_site
   Liquid Exception: undefined method `[]' for nil:NilClass in 2011-05-23-kitten.md
   /tmp/build_g6oesylzhptw/_plugins/flickr_photo.rb:47:in `render'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/liquid-2.4.1/lib/liquid/block.rb:94:in `block in render_all'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/liquid-2.4.1/lib/liquid/block.rb:92:in `collect'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/liquid-2.4.1/lib/liquid/block.rb:92:in `render_all'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/liquid-2.4.1/lib/liquid/block.rb:82:in `render'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/liquid-2.4.1/lib/liquid/template.rb:124:in `render'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/liquid-2.4.1/lib/liquid/template.rb:132:in `render!'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/jekyll-0.12.0/lib/jekyll/convertible.rb:79:in `do_layout'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/jekyll-0.12.0/lib/jekyll/post.rb:195:in `render'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/jekyll-0.12.0/lib/jekyll/site.rb:200:in `block in render'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/jekyll-0.12.0/lib/jekyll/site.rb:199:in `each'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/jekyll-0.12.0/lib/jekyll/site.rb:199:in `render'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/jekyll-0.12.0/lib/jekyll/site.rb:41:in `process'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/gems/jekyll-0.12.0/bin/jekyll:264:in `<top (required)>'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/bin/jekyll:19:in `load'
   /tmp/build_g6oesylzhptw/vendor/bundle/ruby/1.9.1/bin/jekyll:19:in `<main>'
   rake aborted!
   Command failed with status (1): [bundle exec jekyll...]
   /tmp/build_g6oesylzhptw/Rakefile:10:in `block (2 levels) in <top (required)>'
```

**Comment by bolderbrooklyn** [comment]
Fixed by 72deee341a600cf3dc681a25bcdddbccfd20a6fb. Apparently environment variables aren't available yet during the `assets:precompile` phase of the Heroku Ruby buildpack.

**Comment by bolderbrooklyn** [comment]
Follow-up: [`user-env-compile`](https://devcenter.heroku.com/articles/labs-user-env-compile) in Heroku Labs makes environment variables available during slug compilation. Reverted 72deee341a600cf3dc681a25bcdddbccfd20a6fb with 07b2fae01de37d9942dacf964f650063f0267281 to go back to using an environment variable (the API key that had been committed is no longer the one used).
